alonet/callbacks/object_detector_callback.py

```diff
@@ -129,8 +130,12 @@ def log_masks(self, frames: list, pred_masks: list, trainer: pl.trainer.trainer.
             A set of predicted segmentation masks
         trainer: pl.trainer.trainer.Trainer
             Lightning trainer
+        name : str
+            Name to show in wandb
         """
         images = []
+        # Show categories with same color if there are many categories
+        color_by_cat = len(frames[0].segmentation.labels.labels_names) > 20
```

Comment:
Sounds arbitrary. Why 20 ?

Reply:
@thibo73800  Yes. It is a value that a choose. For datasets with high amount of  objects type  (like COCO), it is better show one color by object id (all `person` with the same color) . But, if we used a finetune model for detect only persons (for example), its better show every person detected by one different color. I though that maybe this process could make automatically, selecting as threshold the value of 20